<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="position1_input">
       <property name="toolTip">
        <string>Начальная позиция коня</string>
       </property>
       <property name="placeholderText">
        <string>b2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="position2_input">
       <property name="toolTip">
        <string>Конечная позиция коня</string>
       </property>
       <property name="placeholderText">
        <string>h6</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="StartButton">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string>Начать движение</string>
       </property>
       <property name="text">
        <string>Старт</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="knight_picture">
    <property name="geometry">
     <rect>
      <x>69</x>
      <y>223</y>
      <width>26</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>knight-icon.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGraphicsView" name="chess_board">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>270</width>
      <height>270</height>
     </rect>
    </property>
    <property name="sceneRect">
     <rectf>
      <x>0.000000000000000</x>
      <y>0.000000000000000</y>
      <width>268.000000000000000</width>
      <height>268.000000000000000</height>
     </rectf>
    </property>
   </widget>
   <zorder>knight_picture</zorder>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>chess_board</zorder>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
